db/migrate/20170911175619_rename_tags_to_concepts.rb

```diff
@@ -0,0 +1,5 @@
+class RenameTagsToConcepts < ActiveRecord::Migration[5.1]
```

Comment:
I know this would be a total pain, but would it be possible to roll back the migrations and consolidate this code down into a single migration?

Reply:
No pain, I meant to consolidate / get rid of the double table rename.